TunnelVisionLabs.ReferenceAssemblyAnnotator/TunnelVisionLabs.ReferenceAssemblyAnnotator.targets

```
@@ -42,7 +42,32 @@
     <AddStandardAssembliesForAnnotationDependsOn>ResolveTargetingPackAssets</AddStandardAssembliesForAnnotationDependsOn>
   </PropertyGroup>
 
-  <Target Name="ResolveAnnotatedReferenceAssemblies">
+  <Target Name="DetermineAnnotationPackageDownloadVersion">          
+    <ItemGroup>
+      <_CandidatePackageDownloadVersion Include="@(PackageDownload->'%(Version)'->Distinct())" Condition="'%(PackageDownload.Identity)' == 'Microsoft.NETCore.App.Ref'" />
```

💭 Does this work with just `%(Identity)`?

```suggestion
      <_CandidatePackageDownloadVersion Include="@(PackageDownload->'%(Version)'->Distinct())" Condition="'%(Identity)' == 'Microsoft.NETCore.App.Ref'" />
```

---

I forgot to try this.